hack/test-integration.sh

```diff
@@ -86,9 +86,15 @@ function exectest() {
 
 	if [[ ${result} -eq 0 ]]; then
 		os::text::print_green "ok      $1"
-		exit 0
+		# Remove the etcd directory to free up space when the test case finished
+		# succesfully.
+		rm -rf /tmp/etcd????????? && exit 0
 	else
 		os::text::print_red "failed  $1"
+		# Archive the etcd directory when the test case failed.
+		mkdir -p /tmp/test-integration/etcd/
+		mv /tmp/etcd????????? /tmp/test-integration/etcd/ &>/dev/null || :
+    exit 0
```

Comment:
will fix it... i hate to configure my vim to treat tabs differently in shells scripts ;-)